yaptide/simulation_runner/shieldhit_runner.py

```diff
@@ -1,49 +1,56 @@
 #!/usr/bin/env python
 
+import sys
 import tempfile
+import celery
 
-from pymchelper.executor.options import SimulationSettings
-from pymchelper.executor.runner import Runner as SHRunner
-from pymchelper.estimator import Estimator
-from pymchelper.page import Page
-from pymchelper.axis import MeshAxis
+celery = celery.Celery("application")
+celery.conf.broker_url = 'redis://localhost:6379/0'
+celery.conf.result_backend = 'redis://localhost:6379/0'
 
-from ..converter.converter.converter import DummmyParser
-from ..converter.converter.converter import Runner as ConvRunner
 
-
-def run_shieldhit(param_dict: dict, raw_input_dict: dict) -> dict:
+@celery.task(bind=True)
+def run_shieldhit(self, param_dict: dict, raw_input_dict: dict) -> dict:
     """Shieldhit runner"""
+    from pymchelper.executor.options import SimulationSettings
+    from pymchelper.executor.runner import Runner as SHRunner
+
+    # dirty hack needed to properly handle relative imports in the converter submodule
+    sys.path.append('yaptide/converter')
+    from ..converter.converter.api import get_parser_from_str, run_parser  # skipcq: FLK-E402
+
     # create temporary directory
     with tempfile.TemporaryDirectory() as tmp_output_path:
 
         # digest dictionary with project data (extracted from JSON file)
         # and generate SHIELD-HIT12A input files
-        conv_runner = ConvRunner(parser=DummmyParser(),
-                                 input_data=raw_input_dict,
-                                 output_dir=tmp_output_path)
+        conv_parser = get_parser_from_str("dummy")
```

Comment:
```suggestion
        conv_parser = get_parser_from_str("shieldhit")
```
https://github.com/yaptide/yaptide/issues/130

In `/sh/demo` route we will still be using "dummy" parser